application_sandbox/overlapping_frames/main.cpp

```diff
@@ -0,0 +1,649 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <utility>
+
+#include "support/containers/unique_ptr.h"
+#include "support/containers/unordered_map.h"
+#include "support/containers/vector.h"
+#include "support/entry/entry.h"
+#include "vulkan_core.h"
+#include "vulkan_helpers/vulkan_application.h"
+#include "vulkan_helpers/helper_functions.h"
+#include "vulkan_helpers/vulkan_model.h"
+#include "vulkan_wrapper/command_buffer_wrapper.h"
+#include "vulkan_wrapper/descriptor_set_wrapper.h"
+#include "vulkan_wrapper/sub_objects.h"
+
+struct CommandTracker {
+    containers::unique_ptr<vulkan::VkCommandBuffer> command_buffer;
+    containers::unique_ptr<vulkan::VkFence> rendering_fence;
+    containers::unique_ptr<vulkan::DescriptorSet> descriptor_set;
+};
+
+uint32_t tri_vert_shader[] =
```

Comment:
Might be nicer to put these into a namespace:

namespace gBuffer {
    const uint32_t vert_shader[] = 
      #include ....
   const uint32_t frag_shader[] = 
     #include ....
}

Then reference as gBuffer::vert

Reply:
Done